From 1d0b5bbd2460f22bcd3626efc4acf282ebf0e1b5 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 20 May 2022 08:22:55 +0300
Subject: [PATCH 41/50] fdts: s32cc: Correct RDC simple_bus_reg warning

This corrects dtc warning:
Warning (simple_bus_reg): /soc/rdc@440080000: simple-bus unit address
format error, expected "40080000"

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32cc.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index a7f0b36a3..1c7143ac9 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -809,7 +809,7 @@
 			clocks = <&clks S32GEN1_SCMI_CLK_SERDES_REF>;
 		};
 
-		rdc: rdc@440080000 {
+		rdc: rdc@40080000 {
 			compatible = "nxp,s32cc-rdc";
 			reg = <0x0 0x40080000 0x0 0x3000>;
 		};
-- 
2.17.1

